docs/topics/spiders.rst

```diff
@@ -145,6 +145,11 @@ scrapy.Spider
            <topics-addons>`. The final settings are available in the
            :meth:`start_requests` method and later.
 
+           It's important to note that the :param crawler: at the time of execution 
+           of spider does not have most of its components fully initialized.
```

Comment:
This is true not "at the time of execution of spider" but specifically inside the `from_crawler()` method.